docs/src/modules/javascript/pages/proto.adoc

```diff
@@ -35,14 +35,25 @@ include::example$js-valueentity-shopping-cart/proto/shoppingcart_api.proto[tag=p
 
 === Messages
 
-You define messages that can be sent to or returned from the service. Each message for an Entity that is input to a RPC command, *must* contain an Entity key. In the example, this includes `AddLineItem`, `RemoveLineItem`, and `GetShoppingCart` where the `user_id` is the `(akkaserverless.field).entity_key`. Akka Serverless uses the `entity_key` to know which instance of an Entity a message is for. If more than one field is specified as an Entity key, the fields are concatenated together. Akka Serverless serializes Entity keys to strings.
+You define messages that can be sent to or returned from the service. Each message for an Entity that is input to a RPC command, *must* contain an Entity key. In the example, this includes `AddLineItem`, `RemoveLineItem`, and `GetShoppingCart` where the `user_id` is the `(akkaserverless.field).entity_key`.
+
+Akka Serverless uses the `entity_key` to know which instance of an Entity a message is for. If more than one field is specified as an Entity key, the fields are concatenated together. Akka Serverless serializes Entity keys to strings.
 
 .proto/shoppingcart_api.proto
 [source,proto]
 ----
 include::example$js-valueentity-shopping-cart/proto/shoppingcart_api.proto[tag=proto_messages]
 ----
 
+To use a multi-field key, add the `entity_key` notation to each field. For example, if the domain proto file `SomeMessage` element defines both `first_field` and `second_field` as part of the key, do the same for the service message:
```

Comment:
The part about defining entity keys for both domain and service messages doesn't make sense here. Entity keys are only defined on incoming messages for the service, so that messages can be routed to the correct entity.

```suggestion
To use a multi-field key, add the `entity_key` notation to each field. For example, if the `SomeMessage` service message defines both `first_field` and `second_field` as part of the key:
```

Reply:
Fixed.